tests/opensource-suites/dash/selectors/datetimeSelectors.test.ts

```diff
@@ -0,0 +1,46 @@
+import {Page} from '@playwright/test';
+
+import DashboardPage from '../../../page-objects/dashboard/DashboardPage';
+import {deleteEntity, openTestPage, slct} from '../../../utils';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+import {Workbook} from '../../../page-objects/workbook/Workbook';
+import {WorkbooksUrls} from '../../../constants/constants';
+
+const expectedValues = [
+    '01.08.2022 14:11:11 - 09.08.2022 15:22:23',
+    '03.09.2022 13:22:11 - 08.09.2022 11:11:11',
+];
+
+datalensTest.describe('Dashboards - Selectors by date/time', () => {
+    datalensTest('Date/time selectors should display the time', async ({page}: {page: Page}) => {
+        const dashboardPage = new DashboardPage({page});
+        const workbookPO = new Workbook(page);
+
+        await openTestPage(page, WorkbooksUrls.E2EWorkbook);
+
+        await workbookPO.createDashboard({
+            editDash: async () => {
+                await dashboardPage.addDateRangeSelector({
+                    controlTitle: 'test-control-1',
+                    controlFieldName: 'test-control-field-1',
+                    range: ['01.08.2022 14:11:11', '09.08.2022 15:22:23'],
```

Comment:
```suggestion
                    range: expectedValues[0].split(' - '),
```
To have one data source
Here and in the second case